### DISCUSSION TITLE
How to switch user with Regolith/Wayland/Sway on Debian

### DISCUSSION THREAD

**Discussion by SimonLefort** [regolith-linux/regolith-desktop#1061] [Q&A]
I just installed a fresh Regolith on Debian (my notes: [Installation de Regolith sur une Debian 12](https://blog.simonlefort.be/regolith-on-debian.html) and [Configuration de Regolith sur une Debian 12](https://blog.simonlefort.be/config-regolith-on-debian.html) ).

I can't find a way to switch user. Can you help me ?

**Comment by SimonLefort** [comment]
Solution : Ctrl + Alt + F1

Not really what I imagined but it works.

**Comment by SimonLefort** [comment]
I found the right solution : **gdmflexiserver**

Thanks to [aquaherd on unix.stackexchange.com](https://unix.stackexchange.com/a/67698).

**Comment by kgilmer** [comment]
Nice, good to know @SimonLefort .  Out of curiosity, what does `gdmflexiserver` provide over the first option you mentioned?

**Reply by SimonLefort** [reply]
It's easier to add to my menu : 

![menu](https://github.com/user-attachments/assets/3e69dc62-45e0-46a2-a7bf-379c61a2ec97)
